modules/dreamview/backend/simulation_world/simulation_world_service.cc

```diff
@@ -813,10 +812,18 @@ void SimulationWorldService::CreatePredictionTrajectory(
   for (const auto &traj : obstacle.trajectory()) {
     Prediction *prediction = world_object->add_prediction();
```

Comment:
The function CreatePredictionTrajectory has a wrong order for input/output variables.

Reply:
done